lib/govspeak.rb

```diff
@@ -161,10 +161,10 @@ def insert_strong_inside_p(body, parser = Govspeak::Document)
       {\/button} # match ending bracket
       (?:\r|\n|$) # non-capturing match to make sure end of line and linebreak
     }x) { |attributes, text, href|
-      button_classes = "button"
-      button_classes << " button-start" if attributes =~ /start/
+      button_classes = "govuk-button"
```

Comment:
Could have considered using `GovukPublishingComponents.render` here instead of as a post process?

Reply:
Possibly, yes. To be honest I wasn't exactly sure how to do that, but that seems sensible. I might have another crack at it another time.

Reply:
I think you'd replace line 176 with a call to the render function

Reply:
e.g.

```ruby
GovukPublishingComponents.render(
  "govuk_publishing_components/components/button",
          text: text.strip,
          href: href.strip,
          start: (attributes =~ /start/),
          data_attributes: {
            module: 'cross-domain-tracking',
            "tracking-code": cross_domain_tracking.strip,
            "tracking-name": 'govspeakButtonTracker',
          },
        )
```

Reply:
Awesome, thanks!

Reply:
Something I forgot in this example, that may be a bug in the existing code.
should only add those data attributes with `if cross_domain_tracking`